### PR TITLE
Added Comments

### PR DESCRIPTION
**Pull request by rjt-gupta**

### CODE REVIEW

File: tanner/emulators/php_object_injection.py [hunk 1]
```diff
@@ -12,6 +12,11 @@ def __init__(self, loop=None):
         self.helper = PHPSandboxHelper(self._loop)
 
     async def get_injection_result(self, code):
+        """
+        Injects the code from attacker to vulnerable code and get emulation results from php sandbox.
+        :param code (str): Input payload from attacker
+        :return: object_injection_result (dict): Keys - file_md5 (md5 hash), stdout (injection result)
```
Comment (afeena) [inline tanner/emulators/php_object_injection.py]: no need to write the name of the returned variable, the type and key description are enough

File: tanner/emulators/xxe_injection.py [hunk 2]
```diff
@@ -29,12 +34,25 @@ def __init__(self, loop=None):
...
         detection = None
         if patterns.XXE_INJECTION.match(value):
             detection = dict(name='xxe_injection', order=3)
         return detection
 
     async def handle(self, attack_params):
+        """
+        Handler of emulator
+        :param attack_params (list): contains dicts as elements with id and value (payload from attacker) as keys
+        :return: dict: value (result of emulator), page (if set to true the payload will be injected to index.html
```
Comment (afeena) [inline tanner/emulators/xxe_injection.py]: `(dict)`
for consistency with your style of naming types :)

File: tanner/emulators/php_object_injection.py [hunk 3]
```diff
@@ -31,12 +36,25 @@ def __init__(self, loop=None):
...
         detection = None
         if patterns.PHP_OBJECT_INJECTION.match(value):
             detection = dict(name='php_object_injection', order=3)
         return detection
 
     async def handle(self, attack_params):
+        """
+        Handler of emulator
+        :param attack_params (list): contains dicts as elements with id and value (payload from attacker) as keys
+        :return: dict: value (result of emulator), page (if set to true the payload will be injected to index.html
```
Comment (afeena) [inline tanner/emulators/php_object_injection.py]: `(dict)`

File: tanner/utils/mysql_db_helper.py [hunk 4]
```diff
@@ -15,15 +15,23 @@ def __init__(self):
...
                                       )
         return conn
 
     async def check_db_exists(self, db_name, ):
-
-        # Checks if DB exists or not, Returns 0 if no such database exists else 1
+        """
+        Checks if DB exists or not
+        :param db_name (str): mysql db name
+        :return: result (integer): 0 if no such database exists else 1
```
Comment (afeena) [inline tanner/utils/mysql_db_helper.py]: `(int)`

File: tanner/utils/mysql_db_helper.py [hunk 5]
```diff
@@ -58,6 +68,11 @@ def __init__(self):
         conn.close()
 
     async def delete_db(self, db):
+        """
+        Deletes the database
+        :param db: db name to be deleted
```
Comment (afeena) [inline tanner/utils/mysql_db_helper.py]: type?

File: tanner/emulators/php_object_injection.py (hunk 1 above)
Comment (afeena) [inline tanner/emulators/php_object_injection.py]: please choose one style for describing keys of the dict or do not name them at all

File: tanner/utils/mysql_db_helper.py [hunk 6]
```diff
@@ -15,15 +15,23 @@ def __init__(self):
...
+        :return: connection object
+        """
+
         conn = await aiomysql.connect(host=TannerConfig.get('SQLI', 'host'),
                                       user=TannerConfig.get('SQLI', 'user'),
                                       password=TannerConfig.get('SQLI', 'password')
                                       )
         return conn
 
     async def check_db_exists(self, db_name, ):
```
Comment (afeena) [inline tanner/utils/mysql_db_helper.py]: could you please remove additional comma in function parameters? :) I don't know why it is here :D

Comment (rjt-gupta) [reply tanner/utils/mysql_db_helper.py]: Maybe its mainly for multiline code? 
https://github.com/mushorg/tanner/pull/315#discussion_r287697507

File: tanner/emulators/php_object_injection.py (hunk 1 above)
Comment (rjt-gupta) [reply tanner/emulators/php_object_injection.py]: Got it.

File: tanner/emulators/xxe_injection.py (hunk 2 above)
Comment (rjt-gupta) [reply tanner/emulators/xxe_injection.py]: Thanks 😅

File: tanner/utils/mysql_db_helper.py (hunk 6 above)
Comment (afeena) [reply tanner/utils/mysql_db_helper.py]: I would say it's a typo

Comment (rjt-gupta) [reply tanner/utils/mysql_db_helper.py]: Okay.

**Pull request merged**